.github/workflows/tests.yml

```diff
@@ -38,59 +38,16 @@ jobs:
         run: |
           make test
 
-      - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
-        with:
-          project_id: ${{ secrets.GCP_PROJECT_ID }}
-          service_account_key: ${{ secrets.GCP_SA_KEY }}
-          export_default_credentials: true
-
-      - name: Run live tests
-        run: |
-          make test-live-cloud
-        env:
-          LIVE_AZURE_CONTAINER: ${{ secrets.LIVE_AZURE_CONTAINER }}
-          AZURE_STORAGE_CONNECTION_STRING: ${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}
-          LIVE_GS_BUCKET: ${{ secrets.LIVE_GS_BUCKET }}
-          LIVE_S3_BUCKET: ${{ secrets.LIVE_S3_BUCKET }}
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        if: matrix.python-version == 3.8
-
       - name: Build distribution and test installation
         shell: bash
         run: |
           make dist
           python -m pip install dist/cloudpathlib-*.whl --no-deps --force-reinstall
           python -m pip install dist/cloudpathlib-*.tar.gz --no-deps --force-reinstall
 
-      - name: Test building documentation
-        run: |
-          make docs
-        if: matrix.os == 'ubuntu-latest' && matrix.python-version == 3.8
-
-      - name: Deploy site preview to Netlify
-        if: matrix.os == 'ubuntu-latest' && matrix.python-version == 3.8
-        uses: nwtgck/actions-netlify@v1.1
-        with:
-          publish-dir: "./docs/site"
-          production-deploy: false
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          deploy-message: "Deploy from GitHub Actions"
-          enable-pull-request-comment: true
-          enable-commit-comment: false
-          overwrites-pull-request-comment: true
-          alias: deploy-preview-${{ github.event.number }}
-        env:
-          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
-          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
-        timeout-minutes: 1
-
       - name: Upload coverage to codecov
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
This used to only upload on `3.8` as well—do we need to keep that check?

Reply:
I've been deliberately loosening this on some projects recently. Codecov will merge the reports together, which means you get better actual coverage on branching code between Python versions, e.g., new things they added to the standard library that older versions need a backport for.

Reply:
🎉  TIL, cool!